### PR TITLE
Use instance roles instead of explicit user creds

### PR DESCRIPTION
**Pull request by jsf9k**
In this pull request I adapt the cyhy-feeds code to use instance roles instead of explicit user creds.  This pull request goes along with cisagov/cyhy_amis#184.

When reviewing, you will want to consider 4604b34 separately from the other changes.  In that particular commit I made a lot of whitespace changes to make the existing code more readable and to make flake8 happier.

Note also that in order to use this pull request you must add an entry `ES_AWS_CONFIG_SECTION_NAME = elasticsearch` to the `DMARC` section of the `cyhy-data-extract.cfg` config file.  Once this pull request is merged, the AWS access keys and AWS secret keys from that config can (and should) be removed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Nice.  This is nice.  🍹 for you @jsf9k

**Comment by jsf9k** [comment]
Since @KyleEvers is the lead on cyhy-feeds, I'm going to wait until he is able to review and  approve before merging.

**Comment by KyleEvers** [comment]
Looks good to me 👍

**Pull request merged**
